v21.1/take-and-restore-locality-aware-backups.md

```diff
@@ -25,31 +25,55 @@ A locality-aware backup is specified by a list of URIs, each of which has a `COC
 The locality query string parameters must be [URL-encoded](https://en.wikipedia.org/wiki/Percent-encoding).
 {{site.data.alerts.end}}
 
-During locality-aware backups, backup file placement is determined by leaseholder placement, as each node is responsible for backing up the ranges for which it is the leaseholder.  Nodes write files to the backup storage location whose locality matches their own node localities, with a preference for more specific values in the locality hierarchy.  If there is no match, the `default` locality is used.
+During locality-aware backups, backup file placement is determined by [leaseholder](architecture/replication-layer.html#leases) placement, as each node is responsible for backing up the ranges for which it is the leaseholder. Nodes write files to the backup storage location whose locality matches their own node localities, with a preference for more specific values in the locality hierarchy. If there is no match, the `default` locality is used.
 
 ## Create a locality-aware backup
 
 For example, to create a locality-aware backup where nodes with the locality `region=us-west` write backup files to `s3://us-west-bucket`, and all other nodes write to `s3://us-east-bucket` by default, run:
 
-{% include copy-clipboard.html %}
+{% include_cached copy-clipboard.html %}
 ~~~ sql
 > BACKUP INTO
 	  ('s3://us-east-bucket?COCKROACH_LOCALITY=default', 's3://us-west-bucket?COCKROACH_LOCALITY=region%3Dus-west');
 ~~~
 
-can be restored by running:
+This backup can be restored by running:
 
-{% include copy-clipboard.html %}
+{% include_cached copy-clipboard.html %}
 ~~~ sql
 > RESTORE FROM ('s3://us-east-bucket', 's3://us-west-bucket');
 ~~~
 
 Note that the first URI in the list has to be the URI specified as the `default` URI when the backup was created. If you have moved your backups to a different location since the backup was originally taken, the first URI must be the new location of the files originally written to the `default` location.
 
+See [Show a node's locality](#show-a-nodes-locality) for guidance on identifying a node's locality to pass in a backup query.
+
 {{site.data.alerts.callout_info}}
 For guidance on connecting to other storage options or using other authentication parameters, read [Use Cloud Storage for Bulk Operations](use-cloud-storage-for-bulk-operations.html).
 {{site.data.alerts.end}}
 
+## Show a node's locality
+
+To determine the locality that a node was started with, run [`SHOW LOCALITY`](show-locality.html):
+
+{% include_cached copy-clipboard.html %}
+~~~sql
+SHOW LOCALITY;
+~~~
+
+~~~
+	locality
++---------------------+
+region=us-east,az=az1
+(1 row)
+~~~
+
+The output shows the locality to which the node will write backup data. One of the single locality key-value pairs can be passed to `BACKUP` with the `COCKROACH_LOCALITY` parameter—for example, `'s3://us-east-bucket?COCKROACH_LOCALITY=region%3Dus-east'`.
```

Comment:
> One of the single locality key-value pairs can be passed to `BACKUP` with the `COCKROACH_LOCALITY` parameter—for example, `'s3://us-east-bucket?COCKROACH_LOCALITY=region%3Dus-east'`.
 
After I read this, I wondered, what if I just pass "az=az1" ? Is that valid?

Reply:
@livlobo Hmmm, yeah — I was just trying to do some more testing around this and kept getting an "invalid for a single BACKUP location" error. (This may have been my setup.) @adityamaru can a user pass any locality tier as the single key=value pair to `COCKROACH_LOCALITY`?

Reply:
yup they are allowed to pass in any single locality KV, az=az1 being one of them.